packages/core/src/extensions/UniqueID/UniqueID.ts

```diff
@@ -309,8 +314,7 @@ const UniqueID = Extension.create({
               });
               return Fragment.from(list);
             };
-            // reset check
-            transformPasted = false;
```

Comment:
are you sure this can be removed? If I see it correctly, `parseFromClipboard` calls `transformPasted`.

This means the behavior of `parseFromClipboard` will now change depending on the value of `transformPasted` (which is determined by what was dropped previously)

Reply:
Yeah I think the only place where it might cause issues is in unit tests, since that's the only time we call `parseFromClipboard` ourselves. Otherwise, it'll get called only after any paste/drop event, where `transformPasted` is always set.

Unit tests pass though so I wouldn't worry about it.

Reply:
Does readd `transformPasted = false;` actually break anything?

If not, I think it's safer to readd it in case we would at some point in the future start calling anything that calls `transformPasted` (like `parseFromClipboard`) without taking UniqueID into account